gdtf-spec.md

```diff
@@ -1547,7 +1555,7 @@ defined XML attributes of a structure geometry are specified in
 | Position                  | [Matrix](#user-content-attrtype-matrix )  | Relative position of geometry; Default value: Identity Matrix           |
 | LinkedGeometry            | [Name](#user-content-attrtype-name )      | The linked geometry.                                                    |
 | StructureType             | [Enum](#user-content-attrtype-enum )      | The type of structure. Defined values are "CenterLineBased", "Detail".  |
-| CrossSectionType          | [Enum](#user-content-attrtype-enum )      | The type of cross section. Defined values are "Fachwerk", "Tube".       |
+| CrossSectionType          | [Enum](#user-content-attrtype-enum )      | The type of cross section. Defined values are "Truss-timbering", "Tube".       |
```

Comment:
Or should we call it simply "bracing"?

Reply:
We renamed it to Truss... https://github.com/mvrdevelopment/gdtf-din-spec/pull/36/commits/4be045a92fee7f6f071e87a7ad979ac6893c4176